tests/integration/components/o-s-s/panel-test.ts

```diff
@@ -0,0 +1,64 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+
+module('Integration | Component | o-s-s/panel', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function (assert) {
+    await render(hbs`
+      <OSS::Panel>
+        <:content>Content</:content>
+      </OSS::Panel>
+    `);
+    assert.dom('.oss-panel').exists();
+    assert.dom('.oss-panel--content').hasText('Content', 'Content should be visible');
+    assert.dom('.oss-panel--footer').doesNotExist('Footer should not be visible');
```

Comment:
What does the doesNotExist parameter do ? 🤔

Reply:
Display an extra message in the test next to `OK` if I remember

Reply:
It shouldn't be in this test but I wanted to check if footer didn't exist based on what named blocks were present in this test case. I test the name blocks further down.